pypeit/wavemodel.py

```diff
@@ -680,7 +680,7 @@ def create_linelist(wavelength, spec, fwhm, sigdetec=2.,
         if True, the file is written in the IRAF format (i.e. wavelength,
         ion name, amplitude).
     vacuum (bool):
-        If True, write the wavelengths in vacuum
+        If True, convert the wavelengths of the created linelist from air to vacuum
```

Comment:
Can we please rename this to something like airtovac. Vacuum is a bad name for this option.